testproject/Assets/Tests/Manual/Scripts/RandomMovement.cs

```diff
@@ -46,7 +46,11 @@ public void Move(int speed)
             else if (!IsServer && IsOwner)
             {
                 // Client must sent Rpc
-                MovePlayerServerRpc(m_Direction * speed);
+                MovePlayerServerRpc(m_Direction * speed*1.05f);
+            }
+            else if (IsServer && !IsOwner)
+            {
+                m_MoveTowardsPosition = Vector3.Lerp(m_MoveTowardsPosition, Vector3.zero, 0.01f);
```

Comment:
I assume this isn't part of the PR?

Reply:
Not really, it was an adjustment I made while testing this PR and is specific to testproject.